### DISCUSSION TITLE
The Wormhole Sent my USDC to a Parallel Universe

### DISCUSSION THREAD

**Discussion by MJSykes888** [wormhole-foundation/wormhole#475] [Developer Q&A] [answered]
To get started using Solana, I first funded a new Solflare Chrome Extension wallet with a bit of SOL (from FTX) and then I tried using wormholebridge.com to transfer USDC from my MetaMask wallet to my Solflare wallet.  I expected to receive wUSDC tokens which I could trade for Solana tokens or use for liquidity mining on Radium or Saber.  Instead, I received "A9mUU4qviSctJVPJdBJWkb28deg915LYJKrzQ19ji3FM" tokens in my Solflare wallet!   When I connect my Solflare wallet to other Solana apps, my SOL balance is detected but not my wUSDC or A9mUU4qviSctJVPJdBJWkb28deg915LYJKrzQ19ji3FM token balance.  Please help me convert my A9mUU4qviSctJVPJdBJWkb28deg915LYJKrzQ19ji3FM tokens into a wrapped USDC token format that can be used on Solana (if such a format exists).  Does "wUSDC" refer to erc20 tokens that were wrapped in some way other than the wormholebridge?  Is there a more popular ETH to Solana bridge?

**Comment by rchristensen** [comment]
Experienced the same issue tonight, tried the new v2 bridge expecting to get USDC SPL same as I have through sollet, instead received USD Coin (Wormhole) which I can only get the wormhole itself to recognize

**Reply by MJSykes888** [reply]
I'd have been less concerned if I received "USDC Wormhole" tokens instead of A9mUU4qviSctJVPJdBJWkb28deg915LYJKrzQ19ji3FM tokens. 
However, I discovered one Solana app that recognizes my A9mUU4qviSctJVPJdBJWkb28deg915LYJKrzQ19ji3FM tokens:  mercurial.finance.

**Reply by rchristensen** [reply]
Thanks for the tip, swapped on mercurial to the regular SPL USDC without issue.

**Reply by quentinadam** [reply]
The token A9mUU4qviSctJVPJdBJWkb28deg915LYJKrzQ19ji3FM is the "weUSDC" token on Saber.so and is tradable for other USD stablecoins.

**Reply by MJSykes888** [reply]
Saber.so will not load for me.  Is it incompatible with the Solflare Chrome Extension wallet?

**Reply by quentinadam** [reply]
Have you tried with Phantom wallet (https://phantom.app/) ?

**Comment by leoluk** [comment, marked as answer]
Like @quentinadam correctly stated above, `A9mUU4qviSctJVPJdBJWkb28deg915LYJKrzQ19ji3FM` is Wormhole-wrapped USDC that can be traded on Saber (and likely others), or sent back to ETH.
